2021-03-29 16:40:53,718 : INFO : <module> : Tkinter version: 8.6
2021-03-29 16:40:53,718 : DEBUG : <module> : All modules imported
2021-03-29 16:40:54,033 : DEBUG : __init__ : Settings loaded
2021-03-29 16:40:54,036 : DEBUG : __init__ : Icons loaded
2021-03-29 16:40:54,173 : DEBUG : __init__ : Theme loaded
2021-03-29 16:40:54,357 : DEBUG : __init__ : Menu created
2021-03-29 16:40:54,358 : DEBUG : __init__ : Right-click menu created
2021-03-29 16:40:54,358 : DEBUG : __init__ : Bindings created
2021-03-29 16:40:54,365 : DEBUG : start_screen : Start screen created
2021-03-29 16:40:54,367 : DEBUG : update_title : update_title: No file open
2021-03-29 16:40:54,367 : DEBUG : update_statusbar : update_statusbar: No file open
2021-03-29 16:40:58,929 : DEBUG : update_statusbar : update_statusbar: No file open
2021-03-29 16:40:58,929 : DEBUG : update_title : update_title: No file open
2021-03-29 16:40:58,929 : DEBUG : mouse : update_title: OK
2021-03-29 16:40:59,090 : DEBUG : update_statusbar : update_statusbar: No file open
2021-03-29 16:40:59,091 : DEBUG : update_title : update_title: No file open
2021-03-29 16:40:59,091 : DEBUG : mouse : update_title: OK
2021-03-29 16:41:01,712 : INFO : exit : Window is destroyed
2021-03-29 16:41:01,723 : INFO : open_file : HexView: opened
2021-03-29 16:41:01,723 : ERROR : open_file : Error when opening file:
Traceback (most recent call last):
  File "/Users/Andy/Documents/Projects/python_projects/PyPlus/pyplus.py", line 606, in open_file
    viewer = ttk.Frame(self.master)
  File "/usr/local/Cellar/python@3.9/3.9.2_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/ttk.py", line 735, in __init__
    Widget.__init__(self, master, "ttk::frame", kw)
  File "/usr/local/Cellar/python@3.9/3.9.2_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/ttk.py", line 552, in __init__
    tkinter.Widget.__init__(self, master, widgetname, kw=kw)
  File "/usr/local/Cellar/python@3.9/3.9.2_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/tkinter/__init__.py", line 2572, in __init__
    self.tk.call(
_tkinter.TclError: NULL main window
